pkg/manager/manager.go

```diff
@@ -67,6 +68,10 @@ func New(configMap string, config *kubevip.Config) (*Manager, error) {
 	adminConfigPath := "/etc/kubernetes/admin.conf"
 	homeConfigPath := filepath.Join(os.Getenv("HOME"), ".kube", "config")
 
+	if config.NodeName == "" {
+		return nil, errors.New("node name is missing from config")
```

Comment:
I haven't tested yet, but wouldn't going back to os.Hostname() restore the previous functionality/behaviour?

Reply:
Yeah, it would. I thought of doing it initially, but I thought I could make the code a bit simpler by omitting it.

Reply:
Done

Reply:
Oops, looks like this has broken the build. `make check` will run all of the linting locally.

Reply:
Yeah, was an embarrassing small error/typo, sorry about that :-)